adapters/sharethrough/sharethrough.go

```diff
@@ -71,5 +112,32 @@ func (a SharethroughAdapter) MakeBids(internalRequest *openrtb2.BidRequest, exte
 		return nil, []error{fmt.Errorf("unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode)}
 	}
 
-	return a.AdServer.responseToOpenRTB(response.Body, externalRequest)
+	var bidReq openrtb2.BidRequest
+	if err := json.Unmarshal(requestData.Body, &bidReq); err != nil {
+		return nil, []error{err}
+	}
+
+	var bidResp openrtb2.BidResponse
+	if err := json.Unmarshal(response.Body, &bidResp); err != nil {
+		return nil, []error{err}
+	}
+
+	bidderResponse := adapters.NewBidderResponse()
+	bidderResponse.Currency = "USD"
+
+	for _, seatBid := range bidResp.SeatBid {
+		for _, bid := range seatBid.Bid {
+			bidType := openrtb_ext.BidTypeBanner
+			if bidReq.Imp[0].Video != nil {
+				bidType = openrtb_ext.BidTypeVideo
+			}
+
+			bidderResponse.Bids = append(bidderResponse.Bids, &adapters.TypedBid{
+				BidType: bidType,
+				Bid:     &bid,
```

Comment:
There's a subtle bug here with exporting a reference to a for loop iterator variable. You should use the index instead,

Please review our updated guidance from the adapter dev guide: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#adapter-code